pkg/sentry/kernel/thread_group.go

```diff
@@ -286,6 +286,16 @@ func (tg *ThreadGroup) SignalHandlers() *SignalHandlers {
 	return tg.signalHandlers
 }
 
+// IsReaperLocked check tg is reaper of the namespace.
+//
+// Preconditions: ts.mu must be locked for writing.
+func (tg *ThreadGroup) IsReaperLocked() bool {
+	if tg.pidns.tgids[tg] == 1 {
```

Comment:
This method could be simplified to:

return tg.pidns.tgids[tg] == 1

Reply:
Thanks, fix code changed in new patch.